source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,124 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v5.37. 
 
 Latest Mattermost Releases:
+- [Release v7.0 - Major Release](#release-v7-0-major-release)
 - [Release v6.7 - Feature Release](#release-v6-7-feature-release)
 - [Release v6.6 - Feature Release](#release-v6-6-feature-release)
 - [Release v6.5 - Feature Release](#release-v6-5-feature-release)
-- [Release v6.4 - Feature Release](#release-v6-4-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
 
+## Release v7.0 - [Major Release](https://docs.mattermost.com/upgrade/release-definitions.html#major-release)
+
+**Release Day: 2022-06-15**
+
+### Important Upgrade Notes
+ - MySQL self-hosted customers may notice the migration taking longer than usual when having a large number of rows in the ``FileInfo`` table. For MySQL, it takes around 19s for a table of size 0.7M rows. For PostgreSQL, it takes around 1.3ms for a table of size 0.7M rows.
+ - All channel header icons registered by plugins were moved to the new App Bar, even if they do not explicitly use the new registry function to render a component there. A new config setting ``EnableAppBar`` enables this functionality. The setting defaults to ``false`` for self-hosted deployments.
```

Comment:
@itao / @agarciamontoro, this description is a bit blurry to me -- it sounds like they are not moved unless this feature is enabled then? But the release note is written in a way that makes it sound more urgent, vs. adding an option to do this.